qemu/target-arm/translate-a64.c

```diff
@@ -2619,9 +2619,9 @@ static void disas_ldst_single_struct(DisasContext *s, uint32_t insn)
         } else {
             /* Load/store one element per register */
             if (is_load) {
-                do_vec_ld(s, rt, index, tcg_addr, MO_TE + scale);
```

Comment:
i suppose that since MO_TE is removed, something elsewhere must be modified too, or this port is incomplete?